### ISSUE TITLE
It doesn't work across apps... (should it?)

### ISSUE THREAD

**Issue by ragcsalo** [adriano-di-giovanni/cordova-plugin-shared-preferences#1]
I thought this should work across apps... but it doesn't. I use the same set in both my Cordova apps, but the key-value pair I create in one app, doesn't exist in the other app. Is there a way to fix this?

**Comment by adriano-di-giovanni** [comment]
Can you give me an example, please?

**Comment by adriano-di-giovanni** [comment]
In the meanwhile, please refer to the [SharedPreferences.getInstance(name)](https://github.com/adriano-di-giovanni/cordova-plugin-shared-preferences#window.plugins.module_SharedPreferences.getInstance) documentation. If you specify the same name for all the apps, preferences should be available to all of them.

**Comment by ragcsalo** [comment]
Thanks for the answer! :-)
Unfortunately this is not the case... I used the same instance (same name) in both apps, but they were handled as separate set of preferences.

I found several documentations and forum posts on this matter. First of all, if you use MODE_PRIVATE when creating the shared preference instance, only the apps that has the same android:sharedUserId value (in the Manifest file) can access that instance. I tried it, but no luck...

There were also MODE_WORLD_READABLE and MODE_WORLD_WRITEABLE as an option but these are now deprecated and can't be used.

I think I will use an online database and some AJAX calls to store and read data over the web, as I can't manage shared preferences to work across my apps. :-/
